src/plugins/list/testmod_list.c

```diff
@@ -51,7 +53,7 @@ static void doTest (void)
 	elektraPluginClose (check, 0);
 
 	succeed_if (plugin->kdbGet (plugin, ks, parentKey) == 1, "kdbget failed");
-	const Key * key = ksLookupByName (ks, "user/tests/list/meta1", 0);
+	const Key * key = ksLookupByName (ks, "user/tests/list/META1", 0);
```

Comment:
Does this test trigger the bug now?

Reply:
Yes, line 27 configures the rename plugin directly to cut a part of the key name ("to/be/cut"), while line 34 configures every plugin with "config/toupper" which is only used by the rename plugin.

Reply:
I added comments for this to the test.